### PR TITLE
Figured out basic study session workflow

### PR DESCRIPTION
**Pull request by tlelievre26**
## What is this PR introducing?

Adds React components to enable basic study session workflow, including a start screen, session screen, and timeline screen

## What components of our system does this change impact? Which core feature is this relevant to?

Mostly front end, once we get the data gathering components working this will also interact with those

## Does this change depend on any other unmerged PRs? Link them here if so.

No

## Pre-commit to-do list

- [x] Does this PR only contain one feature that cannot be broken up into smaller components?
- [ ] Have you written unit/integration tests for this feature?
- [ ] Have you run the linter over your code for this change?
- [ ] Have you run your changes against the testing suite?
- [x] Have you given a written explanation of the change?
- [x] Have you asked a teammate for review?

### CODE REVIEW

File: src/study-session/session-control.js
Comment (stephzhan) [inline src/study-session/session-control.js:1]: Is this file still just for tracking the directory structure?

File: src/timeline/components/TimelineScreen.jsx
Comment (stephzhan) [inline src/timeline/components/TimelineScreen.jsx:1]: What was the purpose originally for BeginSessionButton being included on the Timeline screen?

File: src/preload.js
Comment (stephzhan) [inline src/preload.js:1]: I assume that this file is going to be used later, but do you have a specific idea in mind w.r.t using preloads?

File: src/study-session/components/BeginSessionButton.jsx
Comment (stephzhan) [inline src/study-session/components/BeginSessionButton.jsx:1]: Is BeginSessionButton going to call the functions that collect & send data? or is it just meant to keep track of the study session timer

**Review by stephzhan — APPROVED** [review]
Nice

**Review by WilsonPrime — APPROVED** [review]
Looks good.

**Pull request merged**